src/precice/Participant.hpp

```diff
@@ -364,20 +569,20 @@ class PRECICE_API Participant {
    * @brief Creates multiple mesh vertices
    *
    * @param[in] meshName the name of the mesh to add the vertices to.
-   * @param[in] positions a span to the coordinates of the vertices
+   * @param[in] coordinates a span to the coordinates of the vertices
    *            The 2D-format is (d0x, d0y, d1x, d1y, ..., dnx, dny)
    *            The 3D-format is (d0x, d0y, d0z, d1x, d1y, d1z, ..., dnx, dny, dnz)
    *
    * @param[out] ids The ids of the created vertices
    *
    * @pre initialize() has not yet been called
-   * @pre position.size() == getMeshDimensions(meshName) * ids.size()
+   * @pre \p coordinates.size() == getMeshDimensions(meshName) * ids.size()
    *
    * @see getDimensions()
    */
   void setMeshVertices(
       ::precice::string_view        meshName,
-      ::precice::span<const double> positions,
+      ::precice::span<const double> coordinates,
       ::precice::span<VertexID>     ids);
```

Comment:
It's a bit arbitrary that we have here `VertexID`, but the single-call version `setMeshVertex` has the return type `int`. A brief explanation of the VertexID type would be necessary and also helpful to better understand what these IDs are and why they are needed.

Reply:
Very good point!